blocks/library/image/index.js

```diff
@@ -50,7 +50,7 @@ registerBlock( 'core/image', {
 		{
 			icon: 'align-center',
 			title: wp.i18n.__( 'Align center' ),
-			isActive: ( { align } ) => 'center' === align,
+			isActive: ( { align } ) => ! align || 'center' === align,
```

Comment:
Probably would be good to get https://github.com/WordPress/gutenberg/pull/812 in, before fixing centering issues.